rebasehelper/plugins/build_log_hooks/files.py

```diff
@@ -202,16 +204,16 @@ def _get_best_matching_files_section(cls, rebase_spec_file, file):
         best_match = ''
         best_match_section = ''
         files = []
-        for sec_name, sec_content in rebase_spec_file.spec_content.sections:
-            if sec_name.startswith('%files'):
-                files.append(sec_name)
-                for line in sec_content:
-                    new_best_match = difflib.get_close_matches(file, [best_match, MacroHelper.expand(line)])
+        for section in rebase_spec_file.spec.sections().content: # pylint: disable=no-member
```

Comment:
Is the pylint disable still necessary when the new context managers are already released?

Reply:
I think so, it's a false positive by pylint.